addon/components/cookie-banner/style.scss

```diff
@@ -0,0 +1,20 @@
+#cookie-banner {
+    width: 100%;
+    position: fixed;
```

Comment:
This does block content on the page, I assume that is expected/fine given that it only shows up until you click accept?